src/js/shield_defs.js

```diff
@@ -2522,6 +2522,16 @@ export function loadShields(shieldImages) {
     },
   };
 
+  // Iran
+  shields["ir:freeways"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white
+  );
+  shields["ir:national"] = roundedRectShield(
```

Comment:
For some reason, 11 of the 69 occurrences of `network=ir:national` are not in Iran but [in Northern Cyprus](https://overpass-turbo.eu/s/1jHc). It may have just been a mistake, judging from [this relation’s history](https://www.openstreetmap.org/relation/11123980/history). I don’t think it should stop us from supporting Iran’s route shields, but maybe we need to run an analysis of where one country’s ISO&nbsp;3166 code is being used as a namespace in another country, intentionally or otherwise.

Reply:
I think that sort of analysis is a good idea. I also noticed `network=AH` on some Asian Highway relations. `AH` is not an ISO country code, but `network=AsianHighway` is probably the better choice between the two. Two-letter values of `network=*` should be reserved for country codes.

Regardless, I've removed the green shields from this PR, leaving just `network=ir:freeways`.